src/handler/post.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+
+import 'dotenv/config';
+import { APIGatewayProxyEvent } from 'aws-lambda';
+import { postTechRecord } from '../services/database';
+import logger from '../util/logger';
+import { generateNewNumber, NumberTypes } from '../services/testNumber';
+
+export const handler = async (
+  event: APIGatewayProxyEvent,
+): Promise<{ body: unknown; statusCode: number }> => {
+  logger.info('Post end point called');
+  try {
+    if (!event.body) {
```

Comment:
Do we not need to validate it's a valid body when there too?

Reply:
Do you mean a valid json? Or valid tech record? If you mean valid tech record, how would I identify what that actually looks like for all of the different vehicle types?

Reply:
I meant a valid tech record. We have the types package for this reason as well no? Should be able to import that, work out vehicle type and call the validator (I imagine skeleton here) to confirm that it's valid. Also shouldn't this handler also at some stage be working out record completeness?

Reply:
Yep, i'll add in the record completeness logic